package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@babel/core": "^7.1.2",
+    "awesome-typescript-loader": "^3.5.0",
```

Comment:
I want to upgrade to webpack4 so we can upgrade this and other packages to their latest packages.